### PR TITLE
Add Easter egg at the Dojo prompt.

### PR DESCRIPTION
**Pull request by sigilante**
It's always irked me that our wordmark `~Urbit` is not valid to type at the Dojo prompt.

Well, no longer is this a problem!

image

Maybe we should UIP this first, but it's a nice little Easter egg.

### CODE REVIEW

**Comment by sigilante** [comment]
The code is a little rough in that it simply adds an alternate parsing route to `~Urbit` that directly invokes the targeted thread.  Since it's a special case, it probably has to/should be handled separately everywhere.

**Comment by jalehman** [comment]
What does it look like constructed out of `~` characters instead? 

Would be nice to see something like this printed at boot too.

**Comment by sigilante** [comment]
https://www.asciiart.eu/image-to-ascii and the SVG files from Other Means

**Comment by sigilante** [comment]
I'd actually like to put in a nice little animation in text but that'll require a bit of Dill work.

File: pkg/arvo/ted/egg.hoon [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+/-  spider
+/+  strandio
+=,  strand=strand:spider
+^-  thread:spider
+|=  arg=vase
+|^
+  =/  m  (strand ,vase)
+  ^-  form:m
+  ;<  =bowl:spider  bind:m  get-bowl:strandio
+  ;<  code=@p  bind:m  (scry:strandio @p /j/code/(scot %p our.bowl))
```
Comment (pkova) [inline pkg/arvo/ted/egg.hoon:14]: Don't need these.

**Comment by sigilante** [comment]
Final version:

```
> -urbit
```

**Review by jalehman — APPROVED** [review]

**Pull request merged**